google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java

```diff
@@ -108,7 +107,11 @@ private Builder(com.google.api.services.bigquery.model.JobConfiguration configur
         this.schema = Schema.fromPb(loadConfigurationPb.getSchema());
       }
       this.ignoreUnknownValues = loadConfigurationPb.getIgnoreUnknownValues();
-      this.projectionFields = loadConfigurationPb.getProjectionFields();
+      if (loadConfigurationPb.getProjectionFields() != null) {
+        this.formatOptions = DatastoreBackupOptions.newBuilder()
```

Comment:
It will override but it is not suppose to happen. The type of `formatOptions` is one of several supported formats so they are mutual exclusive.